sql/item_mods.sql

```diff
@@ -24499,11 +24513,20 @@ INSERT INTO `item_mods` VALUES (20686,525,3);     -- Augments Convert 1:3
 INSERT INTO `item_mods` VALUES (20687,355,45); -- Burtgang iLvL 119 - Atonement
 INSERT INTO `item_mods` VALUES (20688,355,46); -- Tizona iLvL 119 - Expiacion
 INSERT INTO `item_mods` VALUES (20688,431,1); -- Tizona iLvL 119 - Additional Effect - scripts\globals\items\tizona.lua
-INSERT INTO `item_mods` VALUES (20689,355,225); -- Almace iLvL 119 - Chant du Cygne
+INSERT INTO `item_mods` VALUES (20689,355,225);   -- Almace iLvL 119 - Chant du Cygne
+INSERT INTO `item_mods` VALUES (20707,28,24);     -- Medeina kilij - "Magic Atk. Bonus"+24
+INSERT INTO `item_mods` VALUES (20707,311,123);   -- Magic Damage+123
+INSERT INTO `item_mods` VALUES (20707,122,10);    -- Blue Magic skill +10
+INSERT INTO `item_mods` VALUES (20707,369,5);     -- "Refresh"+1
 INSERT INTO `item_mods` VALUES (20715,25,15);     -- Acclimator (iLvl 119): Accuracy +15
 INSERT INTO `item_mods` VALUES (20715,30,15);     -- Magic Accuracy +15
 INSERT INTO `item_mods` VALUES (20715,175,10);    -- Skillchain Bonus +10%
 INSERT INTO `item_mods` VALUES (20715,311,96);    -- Magic Damage +96
+INSERT INTO `item_mods` VALUES (20716,8,12);      -- Perfervid sword - STR+15
+INSERT INTO `item_mods` VALUES (20716,54,25);     -- [Element: Fire]+25
+INSERT INTO `item_mods` VALUES (20716,25,25);     -- Accuracy+20
+INSERT INTO `item_mods` VALUES (20716,32,15);     -- Fire elemental "Magic Atk. Bonus"+15
+INSERT INTO `item_mods` VALUES (20716,431,1);     -- Additional effect: Fire damage - scripts\globals\items\perfervid_sword.lua
```

Comment:
Please don't direct ppl to individual item scripts anymore. soon as I stop having family emergencies I am globalizing additional effect items so that all you have to do is add mods for which type it is. 😄